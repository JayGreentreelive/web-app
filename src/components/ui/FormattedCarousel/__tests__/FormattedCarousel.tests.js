import React from 'react'
import {Block, Carousel} from '@christfellowshipchurch/web-ui-kit'
import renderer from "react-test-renderer"

describe("ContentBlock Renderer", () => {
  it("Displays carousel", () => {
    const tree = renderer.create(
     <div className="container">
      <div className="row ">
        <div className="col">
          <Carousel>
              <Block
                  layout='default'
                  media={{
                    imageUrl: '',
                    imageAlt: ''
                  }} >
                  <Block.Title/>
                  <Block.Subtitle/>
                  <Block.Body/>
              </Block>
          </Carousel>
        </div>
      </div>
    </div>
    )
    expect(tree).toMatchSnapshot()
  })
})
