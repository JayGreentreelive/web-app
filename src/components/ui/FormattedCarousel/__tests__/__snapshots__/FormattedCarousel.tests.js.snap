// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBlock Renderer Displays carousel 1`] = `
<div
  className="container"
>
  <div
    className="row "
  >
    <div
      className="col"
    >
      <div
        className="carousel slide"
        data-ride="carousel"
      />
    </div>
  </div>
</div>
`;
